MapboxNavigation/NavigationViewController.swift

```diff
@@ -233,18 +256,17 @@ open class NavigationViewController: UIViewController {
             return map.view.constraintsForPinning(to: parent.view)
         }
         
-
+        //Manually update the map style since the RMVC missed the KVO "map style change" message
```

Comment:
Are you referring to the key-value observation that gets set up (as `styleObservation`) in `viewDidLoad()`? That’s basically just a “did finish loading map style” hook.

Reply:
As for Style changing, StyleManager gets set up before RouteMapViewController, but is the issue that StyleManager applies a style before RouteMapViewController is added to the view hierarchy?

Reply:
Posterity: Talked about this with Minh, no action needed.